app/assets/javascripts/listings/components/listing/propertyHero.js.coffee

```diff
@@ -33,6 +33,7 @@ angular.module('dahlia.components')
         # right now it's just an array of one
         [listing.imageURL]
 
+      # This will likely have to change.
```

Comment:
nit: Mind updating this comment with when and why this would have to change?

Reply:
looks like it's no longer in use. I'll remove it.